packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/components/TransactionsCompletedRowTransferAccountDetails/TransactionsCompletedRowTransferAccountDetails.tsx

```diff
@@ -13,7 +13,10 @@ const TransactionsCompletedRowTransferAccountDetails: React.FC<TProps> = ({ acco
     const wallet = accounts.wallets?.find(account => account.loginid === loginid);
     const dtradeAccount = accounts.dtrade?.find(account => account.loginid === loginid);
     const dxtradeAccount = accounts.dxtrade?.find(account => account.login === loginid);
-    const mt5Account = accounts.mt5?.find(account => account.login === loginid);
+    // TODO: remove the `replace` calls once backend resolves `statement` and `mt5_login_list` accounts `loginid` inconsistency
+    const mt5Account = accounts.mt5?.find(
+        account => account.login?.replace(/^\D+/g, '') === loginid.replace(/^\D+/g, '')
```

Comment:
whats the source of inconsistency? can it cause incorrect match? (as effectively this removes parts of loginid, so in a way, it makes it "easier" to match)

Reply:
There's a thread for context in Slack, but in short, BE _right now_ always sends those loginids as real, so if there's a `MTD12345` account, in `statement` it's `MTR12345`. The backend will fix this, then we remove this code